tictactoe.py

```diff
@@ -61,20 +65,18 @@ def make_move(self, difficulty='easy', player=None):
                     p = random.randrange(9)
                 self.config = self.config[:p] + player + self.config[p + 1:]
 
-    def print(self):
+    def print(self, config):
```

Comment:
you could set the default parameter of the config parameter to self.config so that you need not unnecessarily pass it everytime

Reply:
I added this for debugging the min_max configs. Will remove.

Reply:
Resolved here: bc6c50cd702c940c930ad708f5e11470987e4fb9